### DISCUSSION TITLE
Some rookie problems

### DISCUSSION THREAD

**Discussion by Misuki743** [thuliteio/doks#1189] [General]
Firstly, thanks you for the maintenance of this project, it is amazing!

When I was trying to set the website, I found that the collapsed navigate bar on the left didn't work and I can't expand it, how do I fix it?

<img width="226" alt="截圖 2024-02-05 下午4 42 45" src="https://github.com/gethyas/doks/assets/92742051/d44190ac-1327-4a80-8600-d2583993179a">
(I turn it off currently, otherwise I can't click files under it)

And another question is that, is it possible to remove the whole "on this page" bar when viewing .md files with title?
<img width="909" alt="截圖 2024-02-05 下午4 49 34" src="https://github.com/gethyas/doks/assets/92742051/e8fa88b3-960a-4c12-ba67-65bddc22cf38">

here is my repository directory: https://github.com/Misuki743/blog

**Comment by h-enk** [comment]
Regarding your second question — just set `toc: false` in the frontmatter of a page

**Comment by h-enk** [comment]
In `config/_default/params.toml` you've set `flexSearch = false` — this also disables the collapsing functionality.

To re-enable the collapsing functionality— just set `bootstrapJavascript = true`
